scripts/provisioning/s3setup_prereqs.json

```diff
@@ -37,7 +37,7 @@
   "init": {
     "rpms": ["symas-openldap", "symas-openldap-servers", "symas-openldap-clients", "haproxy", "cortx-motr", "python36-ldap"],
     "pip3s": ["s3confstore", "cryptography", "s3cipher"],
-    "services": ["sshd", "haproxy", "slapd", "rsyslog"],
+    "services": ["sshd", "haproxy", "slapd", "rsyslogd"],
```

Comment:
pre-requisite for slapd might fail for remote ldap right?

Reply:
validation will not run for remote node